release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -469,6 +469,12 @@ The following metric has been removed:
 
 * `ovnkube_master_skipped_nbctl_daemon_total`
 
+[id="ocp-4-12-multi-zone-ipi-vsphere-installation"]
+==== Multi-zone Installer Provisioned Infrastructure VMware vSphere installation (Technology Preview)
+Beginning with {product-title} {product-version}, the ability to configure multiple datacenter, single vCenter installations using installer-provisioned infrastructureis not available as a Technology Preview feature. Using vCenter tags, you can use this feature to associate vCenter datacenters and compute clusters with {product-title} regions and zones. These associations can help separate failure domains.
```

Comment:
Some typos here. Note that openshift-regions and openshift-zones are special categories in vCenter and special labels in OpenShift.

Current line:
...the ability to configure multiple datacenter, single vCenter installations using installer-provisioned infrastructureis not available as a Technology Preview feature. Using vCenter tags, you can use this feature to associate vCenter datacenters and compute clusters with {product-title} regions and zones. These associations can help separate failure domains.

Suggested change:
...the ability to configure multiple vCenter datacenters and multiple vCenter clusters in single vCenter installation using installer-provisioned infrastructure is now available as a Technology Preview feature. Using vCenter tags, you can use this feature to associate vCenter datacenters and compute clusters with openshift-regions and openshift-zones. These associations can help separate failure domains.

Reply:
This is done. Thank you for your feedback!